larpandoracontent/LArControlFlow/MasterAlgorithm.cc

```diff
@@ -380,8 +381,14 @@ StatusCode MasterAlgorithm::TagCosmicRayPfos(const PfoToFloatMap &stitchedPfosTo
 
     for (const Pfo *const pPfo : nonStitchedParentCosmicRayPfos)
     {
-        if (ambiguousPfos.end() == std::find(ambiguousPfos.begin(), ambiguousPfos.end(), pPfo))
+        const bool isClearCosmic(ambiguousPfos.end() == std::find(ambiguousPfos.begin(), ambiguousPfos.end(), pPfo));
+
+        if (isClearCosmic)
             clearCosmicRayPfos.push_back(pPfo);
+    
+        PandoraContentApi::ParticleFlowObject::Metadata metadata;
+        metadata.m_propertiesToAdd["IsClearCosmic"] = isClearCosmic;
```

Comment:
Implicit bool to float conversion here. Might want to make that explicit, or use conditional operator ( ? : )